pkgs/tools/misc/ncdu/default.nix

```
@@ -24,6 +21,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ ncurses ];
 
+  preInstall = ''
+    export PREFIX=$out
```

It's nicer to support weird edge-cases :)

```suggestion
     export PREFIX="$out"
```